source/docs/03-metadata.xml

```diff
@@ -1444,6 +1444,82 @@
+                     <p>
+                        <figure>
+                           <head/>
+                           <egXML xmlns="http://www.tei-c.org/ns/Examples" rend="code" xml:space="preserve"><xi:include xmlns:xi="http://www.w3.org/2001/XInclude" href="../examples/header/header-sample085.txt" parse="text"/></egXML>
+                        </figure>
+                     </p>
+                     <p><specList>
+                        <specDesc key="dimensions"/>
+                     </specList></p>
+                     <p>The <gi scheme="MEI">dimensions</gi> contain information about the physical size of a source or an entity in general. Usually the dimensions are represented by numerical data. The elements <gi scheme="MEI">height</gi>, <gi scheme="MEI">width</gi>, <gi scheme="MEI">depth</gi>, and <gi scheme="MEI">dim</gi> are available for circumstances that require the capture of the individual dimensions of an object. Do not confuse this element with the <gi scheme="MEI">extent</gi> element, which is used to indicate the quantity of described materials.</p>
+                     <p>
+                        <figure>
+                           <head/>
+                           <egXML xmlns="http://www.tei-c.org/ns/Examples" rend="code" xml:space="preserve"><xi:include xmlns:xi="http://www.w3.org/2001/XInclude" href="../examples/header/header-sample086.txt" parse="text"/></egXML>
+                        </figure>
+                     </p>
+                     <p><specList>
+                        <specDesc key="dim"/>
+                     </specList></p>
+                     <p>An other way of encoding is to use the element <gi scheme="MEI">dim</gi>, that can be used to represent any single dimensional specification. The attribute <att>form</att> is required. When appropriate the elements <gi scheme="MEI">height</gi>, <gi scheme="MEI">width</gi>, and <gi scheme="MEI">depth</gi> are preferred to specify the content of <gi scheme="MEI">dim</gi>.</p>
+                     <p>
+                        <figure>
+                           <head/>
+                           <egXML xmlns="http://www.tei-c.org/ns/Examples" rend="code" xml:space="preserve"><xi:include xmlns:xi="http://www.w3.org/2001/XInclude" href="../examples/header/header-sample087.txt" parse="text"/></egXML>
+                        </figure>
+                     </p>
+                     <p><specList>
+                        <specDesc key="watermark"/>
+                     </specList></p>
+                     <p> Before starting to describe and encode watermarks, it is highly recommended to read up on filigranology separately. It is important to distinguish whether the watermark to be described consists of only one sign or is composed of main and countermarks. The positioning of the watermark on the manuscript page, the completeness as well as the periods of production and use also play a role.</p>
```

Comment:
```suggestion
                     <p> Before starting to describe and encode watermarks, it is highly recommended to read up on filigranology separately. It is important to distinguish whether the watermark to be described consists of only one sign or is composed of main and countermarks. The completeness of the watermark, its positioning on the manuscript page, and the periods of its production and use also play a role.</p>
```

Reply:
This is just to make the sentence flow. 

I would also change "periods" to "period", because I think it would be understood that if one period-of-use played a role, other periods would too. 

And I would remove the word "production" -- I think "use" is sufficient, but maybe there is a reason in the Wasserzeichenwelt for including that.

Reply:
Thank you @mss2221 for your suggestion. I like it. If @gucl-mu also approves I'll merge it.

Reply:
I have tried to differentiate between the production and the use of the paper the watermark is in.. sorry that was not clear enough

Reply:
Ok, I read that as being about the production and use of the watermark, not the paper. If that distinction is necessary, then how about this:

"The completeness of the watermark, its positioning on the manuscript page, and the time period of the paper's production and use also play a role."

(I know the original has "periods", but the singular form of the word can easily refer to discontinuous date ranges, if that is the meaning.)